onnxruntime/test/testdata/identity_dml.onnx

```diff
@@ -0,0 +1,15 @@
+	:�
```

Comment:
consider use a simple onnx python script to generate this test case, instead of check in onnx file.

Reply:
That's right, dropped this one.